src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball {
+    private Colors color = new ColorSupplier().getRandomColor();
+    private int number = new Random().nextInt(100);
```

Comment:
randomize those values in Lottery, not here
Ball class should only define the structure. And lottery would create what you need